src/org/zaproxy/zap/extension/pscanrulesBeta/ZapAddOn.xml

```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Passive scanner rules (beta)</name>
-	<version>14</version>
```

Comment:
Thanks for pointing that out. I checked the version through the market place within the ZAP and it was showing me `14`. Probably it shows the version that is installed. I'll use this link in future for checking the releases.